### PR TITLE
Added ConsoleBuilds folder containing main.cpp

### PR DESCRIPTION
**Pull request by zvan92**
How dis?

### CODE REVIEW

File: ConsoleBuilds/Source/main.cpp [hunk 1]
```diff
@@ -0,0 +1,4 @@
+main()
```
Comment (jrlanglois) [inline ConsoleBuilds/Source/main.cpp]: So, no return type?

Comment (jrlanglois) [reply ConsoleBuilds/Source/main.cpp]: Did you try compiling and running?

Comment (zvan92) [reply ConsoleBuilds/Source/main.cpp]: ^ code::blocks allows me to run main without giving it a return data type... want me to add?

Comment (zvan92) [reply ConsoleBuilds/Source/main.cpp]: IDE was behaving strangely, likely because standard 'main' code was excluded... i'll fix

**Comment by zvan92** [comment]
Changes have been made to main.cpp, including data type identifier and return value

**Comment by zvan92** [comment]
Can you view project files now? code::blocks allowed me to simply create one .cpp file, not sure what else needs to be around it

**Pull request closed without merging**